xbmc/addons/interfaces/AddonBase.cpp

```diff
@@ -192,39 +192,39 @@ void Interface_Base::addon_log_msg(void* kodiBase, const int addonLogLevel, cons
       break;
   }
 
-  CLog::Log(logLevel, "AddOnLog: %s: %s", addon->Name().c_str(), strMessage);
+  CLog::Log(logLevel, "AddOnLog: {}: {}", addon->Name(), strMessage);
 }
 
 bool Interface_Base::get_setting_bool(void* kodiBase, const char* id, bool* value)
 {
   CAddonDll* addon = static_cast<CAddonDll*>(kodiBase);
   if (addon == nullptr || id == nullptr || value == nullptr)
   {
-    CLog::Log(LOGERROR, "Interface_Base::%s - invalid data (addon='%p', id='%p', value='%p')",
+    CLog::Log(LOGERROR, "Interface_Base::{} - invalid data (addon='%p', id='%p', value='%p')",
```

Comment:
Why does `%p` stay old style?

Reply:
Was not 100% sure it makes in this case sense to see better it goes about a pointer, seen also some parts now where not use the `static_cast<void*>` for them an want to change.

Reply:
Seen during test it is not only a can be with `{}`, it is a must be then.
This was given on test:
```
ERROR: Interface_Base::get_setting_bool - invalid data (addon='%p', id='%p', value='%p')
```
Only the `get_setting_bool` was correct the rest has written `%p` to log.